Fitness/Models/ProBio.swift

```diff
@@ -0,0 +1,139 @@
+        
+        let proJuan = ProBio(id: "test", name: "Juan Garcia", profilePic: "juan", secondaryProfilePic: "juanSecondary", bio: "My name is Juan Garcia and I’m a junior studying Operations Research and Information Engineering with a minor in Business. I’m from Edinburg, Texas which is a border town alongside the Rio Grande River which separates Texas and Mexico. I began my lifting career the summer after high school when I was here at Cornell participating in the Prefreshman Summer Program.(shoutout to my 2016 PSP class!) Outside of classes and lifting, I enjoy playing a variety of sports, more importantly basketball, hiking when it’s a beautiful day outside, playing video games, and collaborating with others on ideas that can potentially make positive impacts on communities.", links: urls, expertise: ["Weightlifting"], routines: [routine1, routine2, routine3], summary: "Weightlifting and strength training")
+        
+        return proJuan
+    }
+    
+    public static func getMadeline() -> ProBio {
+        var urls: [PersonalLinkObject] = []
+        urls.append(PersonalLinkObject(url: "https://www.instagram.com/trainwithmadeline/", site: .instagram))
+        
+        let routine1 = ProRoutine(title: "Make time for yourself", routineType: .mindfullness, text: "Give yourself 30 minutes a day to do something that makes you happy. I wake up earlier than I need to every morning for an oat milk latte and 30 minutes to lightly stretch and go through my Instagram feed--just to have some time to myself!")
+        
+        let routine2 = ProRoutine(title: "Enroll in a fitness class", routineType: .cardio, text: "You're already probably getting your steps in without even realizing it (you can thank all the Cornell hills for that!). So to step it up a notch I'd recommend a group fitness class like Spinning for beginners so they can increase their cardio performance with some guidance!")
+        
+        let routine3 = ProRoutine(title: "Start small", routineType: .strength, text: "Start small, stay consistent and pick something you actually like! My advice would be to pick a skill that you'd like to improve and incorporate it consistently at the end of your workouts or throughout the day. When I first started out I really wanted to get better at pushups so I always ended my workouts with 10 pushups, now it's part of my warm up instead!")
+        
```

Comment:
there shouldn't be this many newlines here or anywhere in this file

Reply:
Understood, will remove those